src/rws_client.cpp

```diff
@@ -644,6 +644,11 @@ std::string RWSClient::getLogText(const bool verbose)
   return ss.str();
 }
 
+std::string RWSClient::getLogTextLatestEvent(const bool verbose)
+{
+  return (log_.size() == 0 ? "Log is empty." : log_[0].toString(verbose, 0));
```

Comment:
I would suggest to return an empty string if the log is empty.

And document that `getLogTextLatestEvent(..)` returns an empty string if the log is empty.

That should be sufficient and an empty string makes sense if there are no log messages.

Reply:
Good idea, I have pushed the changes.